governance/polls/Reserve Governance Facilitator Apppointment - August 21, 2023.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Reserve Governance Facilitator Apppointment - August 21, 2023
+summary: Signal your support or opposition to appointing the following candidates as Reserve Governance Facilitator(s).
+discussion_link: https://forum.makerdao.com/t/poll-notification-selection-of-reserve-governance-facilitators/21671
+parameters:
+  input_format: 
+    type: choose-free
+    options: [1,2,3,4]
+    abstain: [0]
+  victory_conditions:
+    - { 
+        type: 'and', 
+        conditions: [
+          { type : approval },
+          { type: majority, percent: 50 },
+        ]
+      }
+    - { type : default, value : 5 }
+  result_display: approval-breakdown
+options:
+   0: Abstain
+   1: Le Bateleur
+   2: VoteWizard
+   3: DAO Masons
+   4: JanSky
+   5: None of the above
+start_date: 2023-08-21T16:00:00
+end_date: 2023-08-24T16:00:00
+---
+# Poll: Reserve Governance Facilitator Apppointment - August 21, 2023
+
+The Governance Facilitators have placed a Governance Poll into the voting system using their powers defined in [MIP113.7.1.4](https://mips.makerdao.com/mips/details/MIP113#7-1-4-1). This Governance [Poll](https://manual.makerdao.com/governance/governance-cycle/weekly-governance-cycle#weekly-governance-cycle-definitions-mip16c1) will be active for three days beginning on Monday, August 21 at 16:00 UTC.
+
+**This is an approval vote.**
+- **You may vote for multiple options.**
+- **You should vot for all options that you would support becoming a Reserve Governance Facilitator.**
+- **Unselected options signal that you do not approve of those options.**
+- **If you do not approve of any of the options then you should selecte "None of the above" as your one and only choice.**
+- **If you wish to abstain, that should be your one and only choice.**
+
+## Note on Display of Winners
+
+Due to limitations of the voting portal frontend, we cannot select multiple winners. However, if up to three candidates receive >50% approval, all of the candidates above 50% will be appointed as Reserve Facilitators, this will be confirmed off-chain in the MakerDAO forum.
```

Comment:
```suggestion
Due to limitations of the voting portal frontend, we cannot display multiple winners if selected. If up to three candidates each receive >50% approval, all of the candidates above 50% will be appointed as Reserve Facilitators, this will be confirmed off-chain in the MakerDAO forum.
```